operator/pkg/ciliumendpointslice/manager.go

```diff
@@ -286,6 +288,16 @@ func (c *cesManagerIdentity) removeCESToIdentity(id int64, cesName CESName) {
 	delete(c.cesToIdentity, cesName)
 }
 
+// initializeMappingForCES overrides the same method on cesMgr and is used to
```

Comment:
Nit: should be `// initializeMappingCEPtoCES overrides ...`

Reply:
:laughing: good catch, thanks!